### ISSUE TITLE
Is there a search for text option?

### ISSUE THREAD

**Issue by Epicity** [reorproject/reor#86]
I can't seem to find a literal word search option in it? I see the AI search which searches on similarity, but I don't think I can trust it. Like if i'm searching for a literal sentence, it should probably be at 100% instead of 75%

Thus the word search

**Comment by samlhuillier** [comment]
Yes this'd definitely be a great feature to add. Would you be interested in opening a PR for this? A good place to start is the lancedb hybrid search docs: https://lancedb.github.io/lancedb/hybrid_search/hybrid_search/

**Comment by subin-chella** [comment]
It looks like the vectordb library we are using in the Reor project doesn't support hybrid search natively. Therefore, I created a custom search. Please check the PR.

> Yes this'd definitely be a great feature to add. Would you be interested in opening a PR for this? A good place to start is the lancedb hybrid search docs: https://lancedb.github.io/lancedb/hybrid_search/hybrid_search/
